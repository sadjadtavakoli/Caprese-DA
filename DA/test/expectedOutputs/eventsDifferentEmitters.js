
#12 function  listner3-test/unit_tests/eventsdifferentemitters.js-12-14 entered throught event connection2 emitted by function eventsDifferentEmitters.js-test/unit_tests/eventsdifferentemitters.js-1-24
#4 function  listner1-test/unit_tests/eventsdifferentemitters.js-4-6 entered throught event connection emitted by function eventsDifferentEmitters.js-test/unit_tests/eventsdifferentemitters.js-1-24
#end Execution