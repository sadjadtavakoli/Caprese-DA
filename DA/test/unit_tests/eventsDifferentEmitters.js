var events = require('events');
var eventEmitter = new events.EventEmitter();

function listner1() {
    //   do nothing 
}

function listner2() {
    //   do nothing 
}

function listner3() {
    // do nothing
}

eventEmitter.addListener('connection', listner1);
eventEmitter.addListener('connection2', listner2);

let eventEmitter2 = new events.EventEmitter();
eventEmitter2.addListener('connection2', listner3);


eventEmitter2.emit('connection2')
eventEmitter.emit('connection');