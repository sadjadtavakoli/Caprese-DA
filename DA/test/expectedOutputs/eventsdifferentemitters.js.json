{"adder-test/unit_tests/eventsdifferentemitters.js-17-19":{"impacted":["t_1","t_49"]},"adder2-test/unit_tests/eventsdifferentemitters.js-20-22":{"impacted":["t_1"]},"adder3-test/unit_tests/eventsdifferentemitters.js-24-26":{"impacted":["t_1","t_42"]},"emitter-test/unit_tests/eventsdifferentemitters.js-28-30":{"impacted":["t_1","t_35","t_38"]},"emitter2-test/unit_tests/eventsdifferentemitters.js-31-33":{"impacted":["t_1","t_21","t_24"]},"keyMap":{"t_1":"test/unit_tests/eventsdifferentemitters.js","t_21":"adder-test/unit_tests/eventsdifferentemitters.js-17-19","t_28":"adder2-test/unit_tests/eventsdifferentemitters.js-20-22","t_35":"adder3-test/unit_tests/eventsdifferentemitters.js-24-26","t_42":"emitter-test/unit_tests/eventsdifferentemitters.js-28-30","t_38":"listner3-test/unit_tests/eventsdifferentemitters.js-13-15","t_49":"emitter2-test/unit_tests/eventsdifferentemitters.js-31-33","t_24":"listner1-test/unit_tests/eventsdifferentemitters.js-5-7"}}